### ISSUE TITLE
Correct way of mapping labels with category_ids

### ISSUE THREAD

**Issue by duklin** [hpanwar08/detectron2#6]
According to the documentation of Detectron2, when registering custom dataset, you should set 'category_id' for each annotation in range [0, num_of_categories) where the value for 'num_of_categories' is reserved for the background class. [link](https://github.com/facebookresearch/detectron2/blob/master/docs/tutorials/datasets.md#register-a-dataset) 

However, the original paper [states](https://github.com/ibm-aur-nlp/PubLayNet/tree/master/pre-trained-models#convert-pre-trained-model-for-fine-tuning-on-another-target-dataset) that the background's id is 0.
 
Please tell me how should I map the labels with the correct category_ids.

**Comment by hpanwar08** [comment]
If you have your  data in coco format then there is no need to map the category_ids. detectron2 with do it internally for you.

**Comment by hungnv21292** [comment]
Hi @hpanwar08 
Thank you for your interesting repo.

I want to fine-tune my dataset that having 4 classes (remove class "title").
Do I need convert pre-trained model as in the [link](https://github.com/ibm-aur-nlp/PubLayNet/tree/master/pre-trained-models) before fine-tunning?

Thanks you very much.
